.gitignore

```diff
@@ -13,3 +13,6 @@ continuous_integration/hdfs-initialized
 .pytest_cache/
 dask-worker-space/
 .vscode/
+*.swp
```

Comment:
This seems fine since we include it for other files, but I typically put these files under a global gitignore.